src/package.coffee

```diff
@@ -114,8 +115,24 @@ class Package
     @menus = []
     @grammars = []
     @settings = []
+    @mainInitialized = false
     @mainActivated = false
 
+  initialize: ->
```

Comment:
I think we should rename this method to `initializeIfNeeded`, just to make it clear that we only call `initialize` once.